Mage.Verify/src/test/java/mage/verify/VerifyCardDataTest.java

```diff
@@ -2371,6 +2378,26 @@ public void checkSplitCyclingAbilities() {
         );
     }
 
+    private String splitCantAttackBlockAlone(String cardName, String refText) {
```

Comment:
I'm not a big fan of this. I think I'd rather leave the text as it is for now, and separately consider if it's possible to handle the abilities as effects that can be combined rather than the current implementation that forces them to be separate abilities.